aikau/src/main/resources/alfresco/core/CoreXhr.js

```diff
@@ -315,6 +315,12 @@ define(["dojo/_base/declare",
                response: response
             }, false, false, requestConfig.alfResponseScope || this.pubSubScope);
          }
+         else if (requestConfig.data && requestConfig.data.alfResponseTopic) {
+            this.alfPublish(requestConfig.data.alfResponseTopic, {
+               requestConfig: requestConfig,
+               response: response
+            }, false, false, requestConfig.data.alfResponseScope || this.pubSubScope);
```

Comment:
The `|| this.pubSubScope` is implicit inside `alfPublish` if an `undefined` publish scope is provided, i.e. this is redundant and could rust.

Additionally, if there was an explicit `responseScope: ""` in the payload, then this will actually override the intended global response scope and use local scope instead, and so would cause incorrect behaviour.